common/src/main/java/org/cloudfoundry/autosleep/config/Config.java

```diff
@@ -113,6 +113,12 @@
         String SECRET = "secret";
 
     }
+    
+    interface OrgEnrollmentParameters {
+        enum EnrolledState {
+            backoffice_enrolled, backoffice_opted_out
+        }
+    }
```

Comment:
Just thinking...maybe we should also move the ServiceInstanceParameters interface (and its enum) out of Config into the new EnrollmentConfig for the same reason.
If we agree, I will submit the change with the next PR.